app/Traits/TranslationTrait.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Traits;
+
+use App\Repositories\SettingsRepository;
+
+trait TranslationTrait
+{
+
+    protected function getText($typeOfText, $text, $replacements = null)
+    {
+        $settingRepository = new SettingsRepository();
+        $lang = $settingRepository->getSetting('localization', 'locale');
+        $messages = json_decode(file_get_contents(prepare_file_path("locale/$lang.json")), true);;
+        $message = $messages[$typeOfText][$text];
+
+        if (!is_null($replacements)) {
```

Comment:
after you'll set the default value for the $replacements arg as empty array - this check will become useless